### PR TITLE
docs: update v8 migration docs

### PR DESCRIPTION
**Pull request by ke1k0**
This is a follow-up update for [Make a table of the deleted props in V8](https://instructure.atlassian.net/browse/INSTUI-2906)  ticket (PR: https://github.com/instructure/instructure-ui/pull/434)

### CODE REVIEW

File: docs/guides/v8-upgrade-guide.md [hunk 1]
```diff
@@ -13,17 +13,46 @@ The main feature for this release is the switch of the theming engine, we replac
...
+  </ToggleBlockquote.Paragraph>
+  <ToggleBlockquote.Paragraph>
+    If you are interested in our detailed reasoning you can read it <Link href="https://gist.github.com/serikjensen/4ba00b653efac1dbf80543c529adabbc">here</Link>.
+  </ToggleBlockquote.Paragraph>
+</ToggleBlockquote>
+```
+
+## Before you upgrade to v8.0
+
+There are a few steps you need to do before you start the migration process. **First, upgrade to the latest v7.x release to have access to all the new components and features.**
```
Comment (matyasf) [inline docs/guides/v8-upgrade-guide.md:38]: I don't what you mean by this part: "to have access to all the new components and features". I'd just write:

"**First, upgrade to the latest v7.x release to be able to follow this guide.**"

File: docs/guides/v8-upgrade-guide.md [hunk 2]
```diff
@@ -13,17 +13,46 @@ The main feature for this release is the switch of the theming engine, we replac
...
+
+You can find the legacy v7 documentation on [legacy.instructure.design](https://legacy.instructure.design).
+
+### Removal of deprecated components and properties
 
-We feel that it is very important that InstUI is easy to use and learn. To accomplish this, we try to use well-known open source solutions whenever possible.
-The previous, custom made theming solution did not fit into this vision (for its defense it was added when there weren't widely used alternatives)
-so we decided to switch to a broadly adopted one. The two main candidates were [emotion.sh](https://emotion.sh) and [styled components](https://styled-components.com), we decided to use emotion.
-If you are interested in our detailed reasoning you can read it [here](https://gist.github.com/serikjensen/4ba00b653efac1dbf80543c529adabbc).
+One of the features of v8.0. is that we deleted all components and properties marked as `deprecated` in v7. Since these are not backwards compatible anymore, please make sure to update everything.
```
Comment (matyasf) [inline docs/guides/v8-upgrade-guide.md:44]: One of the *changes* of v8.0

**Review by matyasf — APPROVED** [review]

File: docs/guides/v8-upgrade-guide.md (hunk 1 above)
Comment (ke1k0) [reply docs/guides/v8-upgrade-guide.md]: I kinda "stole it" from the v7 migration guide, and also, we have a new component (TreeBrowser.Node), but probably your version works well too. Changed it.

**Review by matyasf — APPROVED** [review]

File: docs/guides/themeable-to-emotion-migration-guide.md [hunk 3]
```diff
@@ -149,6 +150,31 @@ If you need to, and your app is using multiply themes, you can specify overrides
 </EmotionThemeProvider>
 ```
 
+#### Mixed and nested overrides
```
Comment (HerrTopi) [inline docs/guides/themeable-to-emotion-migration-guide.md:153]: Thanks, nice work and description :)

**Review by HerrTopi — APPROVED** [review]

**Pull request merged**